Cargo.toml

```diff
@@ -0,0 +1,2 @@
+[workspace]
```

Comment:
uber-nit: maybe we can remove this cargo.toml here. If a workspace is needed (looks yes!), feel free to introduce it in a separate PR!

Reply:
Yeah, deleted it...